tests/e2e/dpa_deployment_suite_test.go

```diff
@@ -359,14 +358,6 @@ var _ = ginkgo.Describe("Configuration testing for DPA Custom Resource", func()
 				DoNotBackupImages:  true,
 			}),
 		}),
-		ginkgo.Entry("DPA CR with unsupportedOverrides", ginkgo.Label("aws", "ibmcloud"), InstallCase{
-			DpaSpec: createTestDPASpec(TestDPASpec{
-				BSLSecretName: bslSecretName,
-				UnsupportedOverrides: map[oadpv1alpha1.UnsupportedImageKey]string{
-					"awsPluginImageFqin": "quay.io/konveyor/velero-plugin-for-aws:oadp-1.4",
-				},
-			}),
-		}),
```

Comment:
Why is this case removed? Is there a replacement?

Reply:
ya.. we'll be consistently running w/ overridden images in other repos and this test conflicts now w/ a users setting or ci setting to override the image.

Reply:
ok cool. A comment in code to say we will have this coverage via other repos would be neat :).